core/lib/statsx.h

```diff
@@ -974,16 +974,28 @@ STATS_DEF("Peak total unallocatable free space", peak_total_wasted_vsize)
 STATS_DEF("Number of unaligned allocations (TEB's etc.)", unaligned_allocations)
 STATS_DEF("Peak unaligned allocations", peak_unaligned_allocations)
 #endif
-RSTATS_DEF("Current vmm blocks for heap", vmm_blocks_heap)
-RSTATS_DEF("Peak vmm blocks for heap", peak_vmm_blocks_heap)
-RSTATS_DEF("Current vmm blocks for cache", vmm_blocks_cache)
-RSTATS_DEF("Peak vmm blocks for cache", peak_vmm_blocks_cache)
+RSTATS_DEF("Current vmm blocks for unreachable heap", vmm_blocks_unreach_heap)
+RSTATS_DEF("Peak vmm blocks for unreachable heap", peak_vmm_blocks_unreach_heap)
 RSTATS_DEF("Current vmm blocks for stack", vmm_blocks_stack)
```

Comment:
Similar to dr_stats_t, is it better to explicitly qualify stack and cache stats identifiers with reach/unreach for clarity. E.g. use vmm_blocks_unreach_stack instead of vmm_blocks_stack. 

Or if brevity is more important, maybe remove the reach/unreach qualifier from both.

Reply:
Originally, my thought was that it would be more confusing to qualify since it would imply there's such a thing as the opposite reachability: hence the names here.  When updating dr_stats_t: I thought a user who does not know that stacks are unreachable and caches are reachable would want the qualification.  I still sort of like that split.  But maybe making them the same is better.  I'll qualify the internal too.